content/hardware/06.nicla/boards/nicla-voice/tutorials/ei-intruder-detector/content.md

```diff
@@ -0,0 +1,481 @@
+---
+title: 'Door Intruder Detector Using ML with the Nicla Voice'
+description: "This application note describes how to identify when intruders are trying to force an entrance door by analyzing surroundings sounds using a Nicla Voice, Machine Learning Tools, and the Arduino IoT Cloud"
+difficulty: intermediate
+tags:
+  - Intruder detector
+  - Nicla
+  - Portenta
+  - BLE
+  - Machine Learning
+  - Edge Impulse
+  - Sound recognition
+  - Arduino IoT Cloud
+  - Application Note
+author: 'Christopher Mendez'
+libraries:
+  - name: ArduinoBLE
+    url: https://reference.arduino.cc/reference/en/libraries/arduinoble/
+  - name: ArduinoIoTCloud
+    url: https://github.com/arduino-libraries/ArduinoIoTCloud
+  - name: Arduino_ConnectionsHandler
+    url: https://github.com/arduino-libraries/Arduino_ConnectionHandler
+software:
+  - ide-v1
+  - ide-v2
+  - arduino-cli
+  - web-editor
+  - iot-cloud
+  - edge-impulse
```

Comment:
Machine Learning Tools instead of Edge Impulse. Here the link if it is useful: https://cloud.arduino.cc/machine-learning-tools/

Reply:
We can respect the metadata but is not a big drama to start a new tag so, changed to machine-learning-tools